core/Common.php

```diff
@@ -543,14 +543,14 @@ public static function getRequestVar($varName, $varDefault = null, $varType = nu
                     $ok    = true;
                 }
             } elseif ($varType === 'integer') {
-                if ($value == (string)(int)$value) {
+                if ($value === (string)(int)$value) {
```

Comment:
Maybe this is causing some issues:
```suggestion
                if ((string)$value === (string)(int)$value) {
```